### ISSUE TITLE
edit article links when downloaded

### ISSUE THREAD

**Issue by meagansiu** [maltyeva/mindimp-pg#69]
fix our pdf links when article is downloaded so that it doesn't say "s3 amazonaws"

**Comment by maltyeva** [comment]
Not sure if this is possible, since all of our PDF/picture resources are stored on Amazon cloud servers (S3). I can generate masked URLs if this is a big issue, but what do you guys think?

**Comment by ChasonMindImp** [comment]
From a branding perspective, a masked URL would look better. It reinforces that the article has become part of our database, that MindImp is giving you this access. 

Though, at this point, that may be low priority.

**Comment by jzthompson** [comment]
Yeah, I agree. It isn't a big priority, but masked URLs would be good to
have eventually.
